packages/pnpm/test/recursive/outdated.ts

```diff
@@ -69,15 +70,61 @@ test('pnpm recursive outdated', async (t: tape.Test) => {
     t.equal(result.status, 0)
 
     t.equal(normalizeNewline(result.stdout.toString()), stripIndent`
-    ┌───────────────────┬─────────┬────────┬──────────────────────┬─────────────────────────────────────────────┐
-    │ Package           │ Current │ Latest │ Dependents           │ Details                                     │
-    ├───────────────────┼─────────┼────────┼──────────────────────┼─────────────────────────────────────────────┤
-    │ is-negative       │ 1.0.0   │ 2.1.0  │ project-2            │ https://github.com/kevva/is-negative#readme │
-    ├───────────────────┼─────────┼────────┼──────────────────────┼─────────────────────────────────────────────┤
-    │ is-negative (dev) │ 1.0.0   │ 2.1.0  │ project-3            │ https://github.com/kevva/is-negative#readme │
-    ├───────────────────┼─────────┼────────┼──────────────────────┼─────────────────────────────────────────────┤
-    │ is-positive       │ 1.0.0   │ 3.1.0  │ project-1, project-3 │ https://github.com/kevva/is-positive#readme │
-    └───────────────────┴─────────┴────────┴──────────────────────┴─────────────────────────────────────────────┘
+    ┌───────────────────┬─────────┬────────┬────────────┬─────────────────────────────────────────────┐
+    │ Package           │ Current │ Latest │ Dependents │ Details                                     │
+    ├───────────────────┼─────────┼────────┼────────────┼─────────────────────────────────────────────┤
+    │ is-negative       │ 1.0.0   │ 2.1.0  │ project-2  │ https://github.com/kevva/is-negative#readme │
+    ├───────────────────┼─────────┼────────┼────────────┼─────────────────────────────────────────────┤
+    │ is-negative (dev) │ 1.0.0   │ 2.1.0  │ project-3  │ https://github.com/kevva/is-negative#readme │
+    ├───────────────────┼─────────┼────────┼────────────┼─────────────────────────────────────────────┤
+    │ is-positive       │ 1.0.0   │ 3.1.0  │ project-1  │ https://github.com/kevva/is-positive#readme │
+    │                   │         │        │ project-3  │                                             │
+    └───────────────────┴─────────┴────────┴────────────┴─────────────────────────────────────────────┘
+    ` + '\n')
+  }
+
+  {
+    const result = execPnpmSync('recursive', 'outdated', '--no-table')
+
+    t.equal(result.status, 0)
+
+    t.equal(normalizeNewline(result.stdout.toString()), stripIndent`
+    is-negative
+    1.0.0 => 2.1.0
+    project-2
+
+    is-negative (dev)
+    1.0.0 => 2.1.0
+    project-3
+
+    is-positive
+    1.0.0 => 3.1.0
+    project-1
+    project-3
```

Comment:
If there will be many (like 50 dependents) then this will be unreadable.
I think at least an explanation should be here.
Something like
```
Dependent: project-1
```

But probably keeping it in one line will fit better into the screen

```
Dependents: project-1, project-2
```

Reply:
Actually, 50 comma-delimited dependents stretched out across multiple lines is completely unreadable. I tried both ways before settling on one per line.

But I would advocate against using a label for dependents only. It will look weird unless all of the items have labels, and at that point you might as well just do `--json`.

Another option would be to put a divider above and below the dependents.